src/main/java/com/conveyal/r5/analyst/broker/Broker.java

```diff
@@ -380,8 +379,27 @@ public void createWorkersInCategory (WorkerCategory category) {
             throw new RuntimeException(e);
         }
 
+        String workerConfigString = cfg.toString();
+        String scriptTemplate;
```

Comment:
This variable leaks out of the only scope it is used in, across a try block. Personally I'd probably put the variable declaration and everything up to req.setUserData(userData); inside the try block.